README.md

```diff
@@ -1,20 +1,38 @@
 <!-- markdownlint-disable MD034 -->
 # Container Storage Interface (CSI) driver for vSphere
 
-This repository provides tools and scripts for building and testing the vSphere CSI provider. This driver is in a stable `GA` state and is suitable for production use. Some of the features may be in the `beta` phase. Please refer [feature matrix](https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/2.0/vmware-vsphere-csp-getting-started/GUID-E59B13F5-6F49-4619-9877-DF710C365A1E.html) for more details.  vSphere CSI driver requires vSphere 6.7 U3 or higher in order to operate.
+The vSphere CSI Driver is a Kubernetes plugin that allows persistent storage for containerized workloads running on vSphere infrastructure. It enables dynamic provisioning of storage volumes and provides features like snapshots, cloning, and dynamic expansion of volumes. The vSphere CSI Driver replaces the [in-tree vSphere storage plugin]( https://kubernetes.io/docs/concepts/storage/volumes/#vspherevolume) and offers integration with vSphere with better scale and performance.
```

Comment:
To be accurate lest call it `vSphere Volume` plugin.

Reply:
done.